photos.css

```diff
@@ -76,16 +76,41 @@ p {
     background: white;
 }*/
 
+#bigdiv img {
+	width: 100%;
+}
+
 #bigdiv {
 	line-height: 170%;
 	-webkit-row-gap:      10px;
 	-webkit-column-count: 3;
 	-webkit-column-gap:   10px;
 	column-count:         3;
-	column-gap:           10px;  
+	column-gap:           10px;
 	row-gap:              10px;
 	padding: 10px;
+}
+
+/* tablet styles */
+@media only screen and (max-width : 800px) {
```

Comment:
every css rule inside this media query is only applied when the screen size is 800px or less.